### PR TITLE
Remove plasma4

### PR DESCRIPTION
**Pull request by kensington**
Just a test (for now)!

### CODE REVIEW

File: profiles/base/package.use.mask [hunk 1]
```diff
@@ -5,6 +5,10 @@
 # This file requires >=portage-2.1.1
 # New entries go on top.
 
+# Michael Palimaka <kensington@gentoo.org (02 Nov 2016)
+# Requires Plasma 4 which is being removed
+net-p2p/ktorrent plasma shutdown
```
Comment (a17r) [inline profiles/base/package.use.mask]: this should be pinned to slot 4 as ktorrent:5 also has USE=shutdown

**Comment by kensington** [comment]
ping

**Comment by a17r** [comment]
looks like drkonqi:4 could also be removed together with these packages.

**Comment by johu** [comment]
Thanks. Updated version based on pkg moves repoman checked and committed.

**Pull request closed without merging**